pkg/kubectl/cmd/config/get_contexts.go

```diff
@@ -89,7 +79,13 @@ func (o *GetContextsOptions) Complete(cmd *cobra.Command, args []string, out io.
 	o.contextNames = args
 	o.out = out
 	o.nameOnly = false
-	if cmdutil.GetFlagString(cmd, "output") == "name" {
+	output := cmdutil.GetFlagString(cmd, "output")
+	supportedOutputTypes := sets.NewString("", "name")
+	if !supportedOutputTypes.Has(output) {
+		fmt.Fprintf(out, "--output %v is not available in kubectl config get-contexts; resetting to default output format\n", output)
+		cmd.Flags().Set("output", "")
```

Comment:
`cmd.Flags().Set("output", "")` looks weird here.  Why not using `cmd.Flags.StringVar()` in L72?

Reply:
this is reset output to ""